src/legacy/core_plugins/kibana/server/tutorials/zeek_logs/index.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n }  from '@kbn/i18n';
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/filebeat_instructions';
+
+export function zeekLogsSpecProvider(server, context) {
+  const moduleName = 'zeek';
+  const platforms = ['OSX', 'DEB', 'RPM'];
+  return {
+    id: 'zeekLogs',
+    name: i18n.translate('kbn.server.tutorials.zeekLogs.nameTitle', {
+      defaultMessage: 'Zeek logs',
+    }),
+    category: TUTORIAL_CATEGORY.SECURITY,
+    shortDescription: i18n.translate('kbn.server.tutorials.zeekLogs.shortDescription', {
+      defaultMessage: 'Collect the logs created by Zeek/Bro.',
+    }),
+    longDescription: i18n.translate('kbn.server.tutorials.zeekLogs.longDescription', {
+      defaultMessage: 'The `zeek` Filebeat module collects the logs from \
+[Zeek](https://www.zeek.org//documentation/index.html). \
+[Learn more]({learnMoreLink}).',
+      values: {
+        learnMoreLink: '{config.docs.beats.filebeat}/filebeat-module-zeek.html',
+      },
+    }),
+    //euiIconType: 'logoZeek',
+    artifacts: {
+      dashboards: [
+        {
+          id: '87b0c430-1a2d-11e9-84b1-a12c578fa9e8',
```

Comment:
This `id` is populated by beats, and the same across all deployments, correct?

Reply:
Yes. I have an update for the zeek dashboard, and this value will change later.

Reply:
You can manually edit the exported dashboard if you want it to have the same ID. So this should be an issue.